scripts/olm-create-bundle.sh

```diff
@@ -10,14 +10,16 @@ ROOT_DIR="$SCRIPTS_DIR/.."
 BUILD_DIR="$ROOT_DIR/build"
 DEFAULT_BUNDLE_CHANNEL="alpha"
 PRESERVE=${PRESERVE:-"false"}
+DEFAULT_OPERATOR_SDK_BIN_PATH="$ROOT_DIR/../../aws-controllers-k8s/code-generator/bin/operator-sdk"
```

Comment:
Bin path refers to a binary here, whereas it refers to the directory in `scripts/install-operator-sdk.sh`. I think it makes more sense to refer to the directory

```suggestion
DEFAULT_OPERATOR_SDK_BIN_PATH="$ROOT_DIR/../code-generator/bin"
```

Reply:
Fixed.